import-service/import/import-products-file.ts

```diff
@@ -0,0 +1,28 @@
+import * as AWSLambda from "aws-lambda";
+import * as AWSSDK from "aws-sdk";
+
+export const importProductsFile = async (event: AWSLambda.APIGatewayEvent) => {
+  const s3 = new AWSSDK.S3({ region: "eu-west-1", signatureVersion: "v4" });
+  try {
+    const { name } = event.queryStringParameters;
```

Comment:
**suggestion**: If we try request this method without querystrings params (https://g8n64lq9sd.execute-api.eu-west-1.amazonaws.com/dev/import for example), labmda return 502 by reason of desctructuring, maybe will be good to exclude this case, what do you think?

Reply:
good point! thanks